bazel/emscripten_toolchain/emdwp-emscripten_bin_linux.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec external/emscripten_bin_linux/bin/llvm-dwp "$@"
```

Comment:
The emdwp.py wrapper should be able to find these files based on LLVM_ROOT, so I don't think you need 5 different wrappers here.

Reply:
I agree, but it doesn't work / or I don't know how to do it, https://github.com/emscripten-core/emsdk/pull/1531#discussion_r1961930494